ResearchKitTests/ORKTextChoiceCellGroupTests.m

```diff
@@ -55,6 +55,19 @@ - (NSArray *)textChoices {
     return choices;
 }
 
+- (NSArray *)textChoicesWithExclusive {
+    static NSArray *choicesWithExclusive = nil;
+    
+    if (choicesWithExclusive == nil) {
+        choicesWithExclusive = @[[ORKTextChoice choiceWithText:@"choice 01" value:@"c1"],
+                                [ORKTextChoice choiceWithText:@"choice 02" value:@"c2" exclusive:YES],
+                                [ORKTextChoice choiceWithText:@"choice 03" value:@"c3"],
+                                [ORKTextChoice choiceWithText:@"choice 04" value:@"c4"]];
```

Comment:
I did modify the example in ORKTest to have multiple exclusive choices, but didn't keep that in there.  I'll add some additional test methods to `ORKTextChoiceCellGroupTests` to verify it works.